sample.html

```diff
@@ -74,10 +74,22 @@
             window.pp.mb.publish(OO.EVENTS.WIKIA.SHOW_AD_FULLSCREEN_TOGGLE, false);
           }
         }, false, this);
+
+        // that's a hack for autoplay on mobile for VPAID ads
```

Comment:
can we mark it as "For the future" or at least mention that vpaids are not fully supported now?